fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/KeyValueCursorTest.java

```diff
@@ -419,4 +420,52 @@ public void emptyScanSplit() {
             return null;
         });
     }
+
+    @Test
+    public void buildWithoutRequiredProperties() {
+        try {
+            RecordCursor<KeyValue> kvCursor = KeyValueCursor.Builder.withSubspace(subspace)
+                                                .build();
+            // Program flow should not reach here.
+            assertTrue(false);
+        } catch (RecordCoreException ex) {
+            assertTrue(true);
+        }
```

Comment:
This `try`-`catch` and the one in `buildWithoutScanProperties` can be replaced with an `assertThrows` statement.

Reply:
DONE